<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShapePopulationViewer</class>
 <widget class="QMainWindow" name="ShapePopulationViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1147</width>
    <height>850</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>ShapePopulationViewer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/Icons/help.png</normaloff>:/Icons/help.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>22</width>
    <height>22</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="BoxControl">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>1091</width>
        <height>200</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1091</width>
        <height>300</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QGroupBox" name="BoxVisualization">
       <property name="geometry">
        <rect>
         <x>710</x>
         <y>30</y>
         <width>181</width>
         <height>251</height>
        </rect>
       </property>
       <property name="title">
        <string>Visualization Options</string>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>161</width>
          <height>95</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QComboBox" name="colorMapBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>Select Color Map</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_flip">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Flip Scalars</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_delete">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Delete Surface</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QGroupBox" name="BoxDisplay">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>30</y>
         <width>181</width>
         <height>261</height>
        </rect>
       </property>
       <property name="title">
        <string>Display Options</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>176</width>
          <height>112</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QRadioButton" name="radioButton_4">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>View all surfaces</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_5">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>View square windows</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="colNumberTXT">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Number of columns :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="colNumberEdit">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QSlider" name="colNumberSlider">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QGroupBox" name="BoxAxes">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>151</width>
         <height>251</height>
        </rect>
       </property>
       <property name="title">
        <string>View Axes</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="1">
         <widget class="QWidget" name="axisButton" native="true">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <widget class="QToolButton" name="toolButton_1">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>20</y>
             <width>31</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>-Z</string>
           </property>
          </widget>
          <widget class="QToolButton" name="toolButton_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>80</y>
             <width>31</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>+Z</string>
           </property>
          </widget>
          <widget class="QToolButton" name="toolButton_4">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>50</y>
             <width>31</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>-X</string>
           </property>
          </widget>
          <widget class="QToolButton" name="toolButton_3">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>50</y>
             <width>31</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>+X</string>
           </property>
          </widget>
          <widget class="QToolButton" name="toolButton_5">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>20</y>
             <width>31</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>+Y</string>
           </property>
          </widget>
          <widget class="QToolButton" name="toolButton_6">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>80</y>
             <width>31</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>-Y</string>
           </property>
          </widget>
          <widget class="QToolButton" name="toolButton_0">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>50</y>
             <width>31</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="BoxSyncho">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>30</y>
         <width>151</width>
         <height>91</height>
        </rect>
       </property>
       <property name="title">
        <string>Synchronisation</string>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>92</width>
          <height>52</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QRadioButton" name="radioButton_1">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Real-time</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_2">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Delayed</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>30</y>
         <width>181</width>
         <height>131</height>
        </rect>
       </property>
       <property name="title">
        <string>Interaction</string>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>101</width>
          <height>88</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QRadioButton" name="radioButton_7">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Datas</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_6">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Move</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_synchro">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Select All</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <zorder>BoxDisplay</zorder>
      <zorder>BoxVisualization</zorder>
      <zorder>BoxAxes</zorder>
      <zorder>BoxSyncho</zorder>
      <zorder>groupBox</zorder>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1147</width>
     <height>500</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>1</horstretch>
      <verstretch>1</verstretch>
     </sizepolicy>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QScrollArea" name="scrollArea">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>1041</width>
         <height>384</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>524287</width>
         <height>524287</height>
        </size>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="widgetResizable">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1094</width>
          <height>475</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_5"/>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1147</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_Open_Directory"/>
    <addaction name="action_Open_Files"/>
    <addaction name="action_Open_Data"/>
    <addaction name="separator"/>
    <addaction name="action_Write_Meshes"/>
    <addaction name="separator"/>
    <addaction name="actionDelete_selected_surfaces"/>
    <addaction name="action_Delete_Surfaces"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionOpenFile">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/Icons/fileopen.png</normaloff>:/Icons/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>Open File...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset>
     <normaloff>:/Icons/print.png</normaloff>:/Icons/print.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>:/Icons/filesave.png</normaloff>:/Icons/filesave.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionFlip_Meshes">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Flip Meshes</string>
   </property>
  </action>
  <action name="action_Write_Meshes">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Write Back Meshes</string>
   </property>
  </action>
  <action name="action_Open_Directory">
   <property name="text">
    <string>Open directory</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset>
     <normaloff>:/Icons/help.png</normaloff>:/Icons/help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionI_3_NIRAL">
   <property name="text">
    <string>I &lt;3 NIRAL</string>
   </property>
  </action>
  <action name="actionAdd_a_Directory">
   <property name="text">
    <string>Add a Directory</string>
   </property>
  </action>
  <action name="actionLoad_vtk_files">
   <property name="text">
    <string>Load new ,vtk files</string>
   </property>
  </action>
  <action name="action_Open_Files">
   <property name="text">
    <string>Open .vtk fIles</string>
   </property>
  </action>
  <action name="action_Open_Data">
   <property name="text">
    <string>Load CSV datas</string>
   </property>
  </action>
  <action name="action_Delete_Surfaces">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Delete all surfaces</string>
   </property>
  </action>
  <action name="actionDelete_selected_surfaces">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Delete selected surfaces</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
